lib/generator/underscore.rb

```diff
@@ -4,6 +4,10 @@ module Underscore
       def underscore
         downcase.gsub(/[- ]/, '_').gsub(/[^\w?]/, '')
       end
+
+      def camel_case
+        underscore.split('_').map(&:capitalize).join
+      end
```

Comment:
* I'm sceptical of any change that is an "add" as part of a 'refactor' commit.
* This change makes the class name `Underscore` inaccurate.